### ISSUE TITLE
Java binding for the libword2vec?

### ISSUE THREAD

**Issue by gembin** [maxoodf/word2vec#6]
I've ever used https://github.com/jnr/jnr-ffi to create Java binding for a c shared library.
Is it possible to create a Java binding for libword2vec?

**Comment by maxoodf** [comment]
Yes, I think it's possible even without a C wrapper. Look at [JavaCPP](https://github.com/bytedeco/javacpp) examples.

**Comment by gembin** [comment]
Yes, I'm aware of JavaCPP, but never tried.  BTW, any plan to provide a C API for libword2vec ?  : -)

**Comment by maxoodf** [comment]
Yes, I thought about it, a C wrapper developing is quite simple, but it makes possible to use the library from Go, Java etc. As soon as I get some free time, I will implement it.

**Comment by gembin** [comment]
That's awesome!
